src/script/view_model/content/ConnectRequestsViewModel.ts

```diff
@@ -17,32 +17,44 @@
  *
  */
 
-import {getLogger} from 'Util/Logger';
+import {getLogger, Logger} from 'Util/Logger';
 import {scrollToBottom} from 'Util/scroll-helpers';
 import {isLastItem} from 'Util/ArrayUtil';
 
 import {ParticipantAvatar} from 'Components/participantAvatar';
+import {MainViewModel} from '../MainViewModel';
+import {ContentViewModel} from '../ContentViewModel';
+import {UserRepository} from '../../user/UserRepository';
+import {ActionsViewModel} from '../ActionsViewModel';
+import {User} from '../../entity/User';
 
-window.z = window.z || {};
-window.z.viewModel = z.viewModel || {};
-window.z.viewModel.content = z.viewModel.content || {};
+type Repositories = {
+  user: UserRepository;
```

Comment:
If we just use this one repository, we should pass it directly and not the whole bunch of repositories in this object.